### PR TITLE
Add version commands

### PR DESCRIPTION
**Pull request by c633**

### CODE REVIEW

File: internal/version.go [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package internal
+
+// Version of executables.
+var Version = "0.1"
```
Comment (arlolra) [inline internal/version.go]: 0.1.0?

Comment (arlolra) [inline internal/version.go]: Or, maybe start at "0.0.0+git" and then the commit that we tag and release will bump the version?

Comment (masomel) [inline internal/version.go]: I like that approach. Then we don't have to worry about manually editing the file every time.

**Comment by arlolra** [comment]
Merged in bc2c84b3

**Pull request closed without merging**